classes/window.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, debug=False):
         Window.init(debug)
 
     @staticmethod
-    def init(debug :bool =False) -> Tuple[int, int, int, int]:
+    def init(debug :bool =False) -> dict:
```

Comment:
```suggestion
    def init(debug :bool =False) -> Dict[int, Tuple[int, int, int, int]]:
```

Introduces `Dict` dependency from `typing`